shanoir-uploader/src/main/java/org/shanoir/uploader/gui/AboutWindow.java

```diff
@@ -21,15 +21,15 @@
 import javax.swing.JLabel;
 import javax.swing.JPanel;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.shanoir.uploader.ShUpConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class AboutWindow extends JFrame {
 
 	private static final Logger logger = LoggerFactory.getLogger(AboutWindow.class);
 
-	String supportMail = "imagerie@ofsep.org";

```

Comment:
Hi @LaurentPV,
we need to keep this for the ofsep-version. We should trigger this with the profile selected by the user.
We might extend the profile for a support mail value and use it to clean this up.
With kind regards, Michael